charts/shoot-core/components/charts/node-problem-detector/templates/vpa.yaml

```diff
@@ -1,9 +1,14 @@
 # TODO(ary92): Delete this Helm chart in a future version.
 ---
-apiVersion: "autoscaling.k8s.io/v1beta2"
+apiVersion: "autoscaling.k8s.io/v1"
 kind: VerticalPodAutoscaler
 metadata:
   name: node-problem-detector
   namespace: kube-system
   annotations:
     resources.gardener.cloud/mode: Ignore
+spec:
```

Comment:
Please do not touch/revert the changes for such files which have `Ignore` mode (they are no longer respected by GRM anyways and will be deleted soon)

Reply:
Thanks, I was wondering about these but forgot to ask, done as suggested.